docs/dictionary/property/layerMode.lcdoc

```diff
@@ -56,8 +56,8 @@ Take a platform game as an example. The main character, any animated elements an
 NON-GAME DRAG EXAMPLE
 Consider a drag-drop interface where a user can 'pick' something up and drop it elsewhere. While the object is being dragged across the stack the <layerMode> can be set to 'dynamic' (particularly important if it has graphic effects applied). Once in its final location the <layerMode> can be returned to 'static'
 
->*Warning:* The <layerMode> only has an effect if the <compositorType property> of the stack is not empty.
+>*Warning:* The <layerMode> only has an effect if the <compositorType> of the stack is not empty.
 
-References: compositorTileSize (property), compositorType property (property), compositorType (property), compositorCacheLimit (property)
+References: compositorTileSize (property), compositorType (property), compositorType (property), compositorCacheLimit (property)
```

Comment:
@livecodeali Derp.  I will fix that.  Does the docs builder get upset by that, by the way?